packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1785,6 +1785,7 @@ describe('shallow', () => {
         const wrapper = shallow(<MyComponent />, { disableLifecycleMethods: true });
         expect(wrapper.find(Table).length).to.equal(0);
         wrapper.instance().componentDidMount();
+
```

Comment:
@ljharb was able to remove the `wrapper.update()` by returning true in the adapter function.

Reply:
This still seems to be failing.